composer.json

```diff
@@ -5,13 +5,13 @@
     "keywords": ["ace", "code", "editor", "symfony", "form", "bundle", "javascript"],
     "license": "MIT",
     "require": {
-        "php": ">=7.3",
+        "php": ">=8.1",
```

Comment:
I would probably change it to `~8.1 || ~8.2 || ~8.3`

Reply:
fixed